soh/include/z64save.h

```diff
@@ -316,6 +316,7 @@ typedef struct {
     /*        */ RandomizerCheck ganonHintCheck;
     /*        */ RandomizerCheck gregCheck;
     /*        */ RandomizerCheck dampeCheck;
+    /*        */ char seed[255];
```

Comment:
There’s nothing currently that enforces the string be 255 characters, do we maybe want to add that limit?